src/invoice2data/main.py

```diff
@@ -26,7 +25,6 @@
 input_mapping = {
     "pdftotext": pdftotext,
     "tesseract": tesseract,
-    "tesseract4": tesseract4,
```

Comment:
Does it make sense to keep `tesseract4` mapped to `tesseract` just for some kind of backward compatibility? I'm not sure if it's worth it, just wondering.

Reply:
In my opinion it's not worth it. was'nt well documented and functional anyway.
There's no active implementation with the old teseract known to me.
We can always add it if we got some feedback from someone who needs it.